.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+name: Release Go Binary
+
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  release-matrix:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        goos:
+          - linux
+          - windows
+          - darwin
+        goarch:
+          - "386"
+          - amd64
+          - arm64
+        exclude:
+          - goarch: "386"
+            goos: darwin
+          # Darwin arm64 fails with /usr/bin/ld: unrecognized option '-pagezero_size'
+          - goarch: arm64
+            goos: darwin
+          - goarch: arm64
+            goos: windows
+    steps:
+      - uses: actions/checkout@v3
+      - uses: wangyoucao577/go-release-action@v1.35
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          goos: ${{ matrix.goos }}
+          goarch: ${{ matrix.goarch }}
+          goversion: https://dl.google.com/go/go1.13.1.linux-amd64.tar.gz
```

Comment:
Is there a reason that this is using a 3-year-old version of the compiler?

Reply:
Nahh that was a copy-pasta.  I'll see what newer releases we could use